### ISSUE TITLE
Add the codes in the C folder.

### ISSUE THREAD

**Issue by rohansaini886** [rohansaini886/Hacktoberfest2022#3]
You can add commit codes in C folder. But Don't commit hello world!.

**Comment by Hussain6** [comment]
Hi Sir can I also contribute to this repository?

**Comment by rohansaini886** [comment]
Yes @SayanTan772 you can contribute your project in this folder. We also want to see your project 😃.

**Comment by Anchit-Nayak** [comment]
Hello can I also contribute towards this project?

**Comment by rohansaini886** [comment]
Yes @Anchit-Nayak You can contribute in this repo.

**Comment by yashpimple** [comment]
@rohansaini886 I have added some code to the C folder please have a look.
